ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -278,52 +292,86 @@ def install(cls, mh):
             setup_dns=False, extra_args=['--no-ntp']
         )
 
-    def test_renew_expired_cert_replica(self):
+    @pytest.fixture
+    def move_date_check_replication(self):
+        # add user to check replication after renewal
+        testuser = 'testuser1'
+        password = 'Secret@123'
+
+        # move system date to expire certs
+        for host in self.master, self.replicas[0]:
+            move_date(host, 'stop', '+3years+1days')
+
+        yield
+
+        # check user on replica
+        stdin = (f"{self.master.config.admin_password}\n"
+                 f"{self.master.config.admin_password}\n"
+                 f"{self.master.config.admin_password}\n")
+        self.master.run_command(['kinit', 'admin'], stdin_text=stdin)
+        tasks.user_add(self.master, testuser, password=password)
+        self.replicas[0].run_command(['kinit', 'admin'], stdin_text=stdin)
+        self.replicas[0].run_command(['ipa', 'user-show', testuser])
+
+        # move date back on replica and master
+        for host in self.master, self.replicas[0]:
+            move_date(host, 'start', '-3years-1days')
+
+    def test_renew_expired_cert_replica(self, move_date_check_replication):
         """Test renewal of certificates on replica with ipa-cert-fix
 
         This is to check that ipa-cert-fix renews the certificates
         on replica
 
         related: https://pagure.io/freeipa/issue/7885
         """
-        move_date(self.master, 'stop', '+3years+1days')
-
         # wait for cert expiry
         check_status(self.master, 8, "CA_UNREACHABLE")
 
         self.master.run_command(['ipa-cert-fix', '-v'], stdin_text='yes\n')
 
         check_status(self.master, 9, "MONITORING")
 
-        # move system date to expire cert on replica
-        move_date(self.replicas[0], 'stop', '+3years+1days')
-
+        # replica operations
         # RA agent cert will be expired and in CA_UNREACHABLE state
         check_status(self.replicas[0], 1, "CA_UNREACHABLE")
 
```

Comment:
Usually it's 'Server-Cert cert-pki-ca' that is in CA_UNREACHABLE state, not the RA